kube-deploy/group_vars/all.yml

```diff
@@ -18,6 +18,8 @@ bootstrap_os: ubuntu
 public_iface: enp0s3
 nat_iface: enp0s3
 proxy_enable: false
+#Only enable this if you are deploying on ARM or ARM64 with Fannel SDN provider
+arm_enabled: false
```

Comment:
@v1k0d3n I was thinking along similar lines originally.  However, I don't have an arm environment I can use to validate that the playbooks are going to run.  I thought that adding it to group_vars would be a good initial approach so that the users can turn it on or off as needed.  I agree that pulling it dynamically is a better approach, I just don't have a way to test it at the moment.  Any ideas?